src/features/basic/rename-tab.ts

```diff
@@ -0,0 +1,14 @@
+function updateTabTitle() {
+  const screenName = document
+    .querySelector(`.${C.ScreenControls.currentScreenName}`)
+    ?.textContent?.trim();
+  document.title = screenName ? `${screenName} - Prosperous Universe` : 'Prosperous Universe';
```

Comment:
This feature doesn't account for notifications. It should include the notification count in parentheses before the tab name.

To get the notification count, use `alertsStore.all.value?.count ?? 0`

Then wrap all of this in `watchEffect` so the title updates automatically on both name and notification count updates.

```suggestion
  watchEffect(() => {
    const screenName = screensStore.current.value?.name;
    const notificationCount = alertsStore.all.value?.count ?? 0;
    document.title = // do your logic here;
  });
```

<img width="260" height="55" alt="image" src="https://github.com/user-attachments/assets/480a3cf5-ba7b-494b-b6e7-d686b7ee4af0" />